broker-core/src/main/java/io/zeebe/broker/exporter/stream/ExporterStreamProcessorState.java

```diff
@@ -89,14 +92,29 @@ public long getPosition(final DirectBuffer exporterId) {
 
   private long getPosition() {
     final DbLong zbLong = exporterPositionColumnFamily.get(exporterId);
-    return zbLong == null ? ExporterRecord.POSITION_UNKNOWN : zbLong.getValue();
+    return zbLong == null ? VALUE_NOT_FOUND : zbLong.getValue();
   }
 
   public void visitPositions(BiConsumer<String, Long> consumer) {
     exporterPositionColumnFamily.forEach(
         (exporterId, position) -> consumer.accept(exporterId.toString(), position.getValue()));
   }
 
+  public long getLowestPosition() {
+    final AtomicLong lowestPosition = new AtomicLong(VALUE_NOT_FOUND);
+    final AtomicBoolean firstIteration = new AtomicBoolean(true);
+
+    visitPositions(
+        (id, pos) -> {
+          if (firstIteration.get() || pos < lowestPosition.get()) {
+            lowestPosition.set(pos);
+            firstIteration.set(false);
+          }
+        });
+
+    return lowestPosition.get();
```

Comment:
To make it more explicitly:

```suggestion
    final LongArrayList positions = new LongArrayList();
    state.visitPositions((exporterId, position) -> positions.addLong(position));
    
    return positions.longStream().min().orElse(-1L);
```